### DISCUSSION TITLE
rotki on windows 7

### DISCUSSION THREAD

**Discussion by roadbridgeaward** [rotki/rotki#5610] [Archive]
Sorry for this question, is there any way to run rotki on Windows 7 ?

**Comment by kelsos** [comment]
Due to Python 3.9 not supporting Windows 7 because it is EOL since 2020 the rotki application should display an unsupported version message. #4722

I am not sure what the status of docker support is on Windows 7, but if Docker still provides Windows 7 compatible builds you could use that instead of the application.

**Reply by roadbridgeaward** [reply]
Thank you so much.
